src/screens/HomeScreen.tsx

```diff
@@ -1,25 +1,69 @@
 import * as React from 'react';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, Dimensions } from 'react-native';
 import { Title } from 'react-native-paper';
-
+import MapView, { Marker } from 'react-native-maps';
+import { getAllTrees } from 'database/firebase';
 import EditScreenInfo from 'components/EditScreenInfo';
 import ViewContainer from 'components/ViewContainer';
+import { ThemeProvider } from '@react-navigation/native';
 
 const styles = StyleSheet.create({
-  separator: {
-    marginVertical: 30,
-    height: 1,
-    width: '80%',
-    backgroundColor: '#eee',
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  map: {
+    width: Dimensions.get('window').width,
+    height: Dimensions.get('window').height,
   },
 });
 
+const isValidLocation = (tree: Tree) => {
+  if (tree.location && tree.location.latitude && tree.location.longitude) {
+    return tree;
+  }
+};
+
 export default function HomeScreen() {
+  const [trees, setTrees] = React.useState<Tree[]>([]);
+
+  React.useEffect(() => {
+    async function getData() {
+      try {
+        const data = await getAllTrees();
+        const validTrees = data.filter(tree => isValidLocation(tree));
+        setTrees(validTrees);
+      } catch (e) {
+        console.warn(e);
+      }
+    }
+    getData();
+  }, [setTrees]);
+
   return (
     <ViewContainer>
-      <Title>Home Screen</Title>
-      <View style={styles.separator} />
-      <EditScreenInfo path="/screens/HomeScreen.tsx" />
+      <View style={styles.container}>
+        <MapView
+          style={styles.map}
+          region={{
+            latitude: 37.733053,
+            longitude: -122.419756,
+            latitudeDelta: 0.00275,
+            longitudeDelta: 0.00275,
+          }}
+        >
+          {trees.map(tree => (
+            <Marker
+              coordinate={{
+                latitude: tree.location.latitude,
```

Comment:
you can do `coordinate={tree.location}` and it will match the keys accordingly no need for redundancy